src/containers/Author.js

```diff
@@ -165,7 +165,7 @@ function mapStateToProps(state, ownProps) {
     title: _.get(authorEntity, 'jobTitle') || '',
     image: authorImage,
     mail: _.get(authorEntity, 'email') || '',
```

Comment:
Note that the email might be empty. Previous approach(algolia-v2) appends the default email(contact@twreporter.org). I suggest we move that fallback here rather than transform the data.

Reply:
Updated.